tests/services/test_services.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020-2021 CERN.
+# Copyright (C) 2020-2021 Northwestern University.
+#
+# Invenio-Drafts-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Service tests.
+
+Test to add:
+- Read a tombstone page
+- Read with missing permissions
+- Read with missing pid
+"""
+
+from datetime import timedelta
+from io import BytesIO
+
+import pytest
+from invenio_db import db
+from mock_module.service import ServiceConfig
+
+from invenio_drafts_resources.services import RecordService
+from invenio_drafts_resources.services.records.tasks import cleanup_drafts
+
+
+#
+# Fixtures
+#
+@pytest.fixture()
+def input_data(input_data):
```

Comment:
Same as for the tasks test file... I think you can simplify a lot if you set files to False and do the tests only for the records.